packages/core/EventManager.ts

```diff
@@ -566,9 +581,7 @@ export default class EventManager {
        *  fallback to the first connected calendar - Shouldn't be a CRM calendar
        */
       // Backwards compatibility until CRM manager is created
-      const [credential] = this.calendarCredentials.filter(
-        (cred) => !cred.type.endsWith("other_calendar") || !cred.type.endsWith("crm")
-      );
+      const [credential] = this.calendarCredentials.filter((cred) => !cred.type.endsWith("other_calendar"));
```

Comment:
Don't we still need to skip all CRM calendar credentials(including those that end in `_crm`) as we are in `createAllCalendarEvents` method

Reply:
We sort the CRM credentials into `crmCredentials` array so they won't be included in the `calendarCredentials` array that's being filtered on.